source/guide/3.2/guide-composer.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: bt_wiki
+title: Blueprint Composer User Guide
+category: Guides
+publish: true
+abstract:
+pageord: 800
+
+
+---
+{%summary%} {{page.abstract}}{%endsummary%}
+
+# Overview
+
+The Blueprint Composer is an editor for composing blueprint yaml files dynamically using a modern Drag and Drop Interface.
+This guide will quickly walk you through the use of the Composer.
+
+{%note title=Note%}
+The [Terminology Reference Page]({{page.terminology_link}}) will help you understand some of the terms stated in this guide.
+For your convenience, links are supplied throughout the guide to point you to the right term definition in the terminology page.
+{%endnote%}
+
+# Composer Components Overview
+
+## Login
+
+### Composer Users
+
+Upon first use of the Blueprint Composer the user will be asked to enter a username.
+After first login, the user will be automatically routed to the Composer UI unless a logout was performed.
+Currently each user can only save one blueprint, modifying that blueprint and saving changes will overwrite that single stored blueprint.
+
+## Composer
+
+### Topology
+
+The topology section allows you to add and remove nodes from the blueprint. Simply choose a node type on the left and drag it to the canvas on the right. This will add the node to the blueprint. To delete it, click on the node to open the properties panel and click on Delete button.
+
+![Blueprint Composer topology](/guide/images/ui/composer/topology.png)
+
+Each of the nodes has an editable name and may contain [properties]({{page.terminology_link}}#properties), [interfaces]({{page.terminology_link}}#interface) and [relationships]({{page.terminology_link}}#relationship-type).
+The Composer interface allows selecting one node at a time, to see the node properties simply click the node and a panel will appear.
+
+![Blueprint Composer topology](/guide/images/ui/composer/sidepane.png)
+
```

Comment:
What im missing here is relationship information.
something like "dragging a node inside another node will create a contained_in relationship", "Connecting a node to another node by dragging from the output nob of a node to an input nob of another node will create a connected_to relationship"
Clicking on the line will show the relationship info and will allow modifying it (types, properties and interfaces)
*will also need to add a picture of those 2 types